__tests__/server/utils/logging/development-formatters/index.spec.js

```diff
@@ -14,50 +14,41 @@
  * permissions and limitations under the License.
  */
 
-import yargs from 'yargs';
-
-jest.mock('yargs', () => ({ argv: {} }));
+jest.mock('../../../../../src/server/utils/logging/development-formatters/verbose', () => 'verbose export');
+jest.mock('../../../../../src/server/utils/logging/development-formatters/friendly', () => 'friendly export');
 
 describe('index', () => {
   let index;
-  let verbose;
-  let friendly;
 
-  function load(logFormat = '') {
+  beforeEach(() => {
     jest.resetModules();
-    yargs.argv = { logFormat };
-    jest.mock('../../../../../src/server/utils/logging/development-formatters/verbose', () => ({
-      formatter: jest.fn(),
-      beforeWrite: jest.fn(),
-      afterWrite: jest.fn(),
-    }));
-    jest.mock('../../../../../src/server/utils/logging/development-formatters/friendly', () => ({
-      formatter: jest.fn(),
-      beforeWrite: jest.fn(),
-      afterWrite: jest.fn(),
-    }));
+  });
 
-    verbose = require('../../../../../src/server/utils/logging/development-formatters/verbose').default;
```

Comment:
`default` is undefined in `verbose.js` and `friendly.js` and it only does `export` not `export default`!